pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,116 @@
+    * The Nagoya Protocol regulates access to genetic resources and conditions for transfer of genetic resources and traditional knowledge across counties. For implementations into national law please consult the [ABS Clearing House](https://absch.cbd.int/en/) (also see [RDMkit compliance monitoring & measurement](compliance))  
+    * Laws & Regulations concerning biosecurity, data export control and national interests might be linked from the [RDMkit national resources pages](national_resources)
+    * In order to ensure conformity with ethical research principles, the following should also be considered for data management:
+        * Follow general research ethics laws and guidelines (see below)
+        * Minimise suffering of animals in research to the absolute minimum, following [the guidelines and laws relevant to your location as a baseline](national_resources). A sound documentation and management of research outputs is an essential cornerstone to reduce unnecessary repetitions of experiments. Consider the use of specialised LIMS systems to capture relevant [metadata](metadata_management)
+        * Reflect on potential future implications of the outcome of research and data capture and sharing for society and environment ({% tool "rri-toolkit" %}, {% tool "rri-self-reflection-tool" %}) involving stakeholders of the research is an important measure in order to receive feedback. If data is presented to stakeholders, this should happen in an accessible format and might require pre-processing, visualisation and guidance.
+        * Transparency and reproducibility of the research project underpin the scientific rigour of the project and reduce unnecessary duplication of efforts. Good RDM following the FAIR principles is a cornerstone of these efforts. Continuous tracking of [provenance](provenance) from e.g. research subjects to samples to data and semantic annotation of processes ([documentation and metadata](metadata_management)) can enhance the trustworthiness and value of research findings.
+        * Automated sharing of research data after a specific period or milestones and deliverables in a project can be a good mechanism to enhance the openness of a project ([data management coordination](dm_coordination))
+​
+
+## How can I identify regulations, guidelines and laws connected to ethics in my research context?
+​
+### Considerations
+* In all cases, your institution’s Data Protection Officer (DPO) is the person to refer to when considering ethical and legal aspects of data management.
+* When looking for recommendations and regulations, it is best to start from the local, that is, starting what is applicable within your discipline, and nationally. Then (if applicable), EU policies, directives and regulations are to be explored, as well as global recommendations (for example, from the UNESCO).
+* The list of resources given below does not attempt to be exhaustive: it tries to highlight the most relevant ones.
```

Comment:
Which list does this refer to? :) maybe you can rewrite this to make it clear, thanks

Reply:
```suggestion
* The solutions given below do not attempt to be exhaustive and highlight only the most relevant ones.
```